docs/pages/en-us/Features.md

```diff
@@ -32,5 +32,8 @@ The service is designed for durability as Elasticsearch provides data redundancy
 
 ## High-Availability
 The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.  
+
+## Audit Web Service API and Client
```

Comment:
In the latest commit I have changed the heading to "Audit Web Service" and removed the mention of RESTful API